.gitignore

```diff
@@ -1,2 +1,4 @@
 .vscode
 /bin
+/**/*.out
```

Comment:
Doesn't `*.out` work just as well?

Reply:
@AlexCombat, I think that `*.out` ignores `.out`-files just in the current directory instead of `/**/*.out` template.